src/Paket.Core/Installation/InstallProcess.fs

```diff
@@ -624,9 +624,16 @@ let InstallIntoProjects(options : InstallerOptions, forceTouch, dependenciesFile
                 with
                 | _ -> ()
 
-/// Installs all packages from the lock file.
 let Install(options : InstallerOptions, forceTouch, dependenciesFile, lockFile : LockFile, updatedGroups, touchedPackages) =
     let root = FileInfo(lockFile.FileName).Directory.FullName
 
-    let projects = RestoreProcess.findAllReferencesFiles root |> returnOrFail
+    let projects = 
+        match RestoreProcess.findAllReferencesFiles root with
```

Comment:
Not: you introduced the returnOrFail function already, can you use it here?

Reply:
Done!